frontend/ui/uimanager.lua

```diff
@@ -1452,9 +1456,9 @@ function UIManager:run()
     -- currently there is no Turbo support for Windows
     -- use our own main loop
     if not self.looper then
-        while self._running do
+        repeat
```

Comment:
I *abhor* `repeat` constructs.

And a `while` makes much more sense here, because we *do* want to check the condition first.

Reply:
I accept you personal decision about repeat-until constructs (as here there is no real performance drawback :) ). (But we set self._running to `true` a few lines above, so repeat-until would be OK.)

Reply:
I'd rather figure out a way to actually *remove* the whole `_running` & `_run_forever` stuff altogether, because it feels a little gadgety ;).